ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/MainActivity.kt

```diff
@@ -117,7 +117,7 @@ fun WearApp(watchRepository: WatchRepository) {
             currentBackStackEntry?.arguments?.getSerializable(SCROLL_TYPE_NAV_ARGUMENT)
                 ?: DestinationScrollType.NONE
 
-        var displayValue by remember { mutableStateOf(2) }
+        var displayValueForUserInput by remember { mutableStateOf(2) }
```

Comment:
Nit: You could make 2  const val at the top... like DEFAULT_USER_INPUT_VALUE or something.

Reply:
same comment above, this will be replaced by viewModel later